test/functional/tests/stats/test_consistency_between_outputs.py

```diff
@@ -0,0 +1,299 @@
+        if sqr_brackets_counter:
+            addition = line[line.index("["):line.index("]") + 1]
+            splitted_line.insert(0, line[:column_width] + addition)
+            splitted_line.insert(1, line[column_width:].replace(addition, ""))
+
+        else:
+            splitted_line.insert(0, line[:column_width])
+            splitted_line.insert(1, line[column_width:])
+
+        # remove whitespaces
+        # save statistic name (with unit) to keys
+        keys.append(re.sub(r'\s+', ' ', splitted_line[0]).strip())
+        # save statistic value to values
+        values.append(re.sub(r'\s+', ' ', splitted_line[1]).strip())
+
+    return keys, values
+
+
+def parse_cache_conf_section(table_as_list):
+    return parse_conf_section(table_as_list, 26)
+
+
+def parse_core_conf_section(table_as_list):
+    return parse_conf_section(table_as_list, 21)
+
+
+def separate_values_to_two_lines(table_as_list, column_width):
+    """
+    If there are two values of the one statistic in different units in one line,
+    replace this line with two lines, each containing value in one unit.
+    """
+    for i, line in enumerate(table_as_list):
+        has_two_units = line.count(" / ")
+        if has_two_units:
+            table_as_list.remove(line)
+            value_parts = line[column_width:].split(" / ")
+
+            table_as_list.insert(i, line[:column_width] + value_parts[0])
+            table_as_list.insert(i + 1, line[:column_width] + value_parts[1])
+
+    return table_as_list
+
+
+def parse_tables_section(table_as_list):
+    """
+    The remaining sections in the 'casadm -P' output have four columns.
+    1st: Usage statistics - statistics names
+    2nd: Count - values dependent on units
+    3rd: % - percentage values
+    4th: Units - full units for values stored in 2nd column
+    """
+    keys = []
+    values = []
+
+    # remove table header - 3 lines, it is useless
+    for _i in range(3):
+        table_as_list.pop(0)
+
+    # remove separator lines, it is also useless
+    for line in table_as_list:
+        if is_table_separator(line):
+            table_as_list.remove(line)
+
+    # split lines to columns by '|' sign and remove whitespaces
+    for line in table_as_list:
+        splitted_line = line.split('|')
+        for i in range(len(splitted_line)):
+            splitted_line[i] = splitted_line[i].strip()
+
+        # save keys and values in order:
+        # key: statistic name and unit
+        # value: value in full unit
+        keys.append(f'{splitted_line[1]} [{splitted_line[4]}]')
+        values.append(splitted_line[2])
+        # key: statistic name and percent sign
+        # value: value as percentage
+        keys.append(f'{splitted_line[1]} [%]')
+        values.append(splitted_line[3])
+
+    return keys, values
+
+
+def is_table_separator(line):
+    """
+    Tables in the 'casadm -P' output have plus signs only on separator lines.
+    """
+    return '+' in line
```

Comment:
this is not always true - depending on OS settings, tables can have different styles:
```
╔══════════════════╤═════════╤═══════╤═════════════╗
║ Usage statistics │  Count  │   %   │   Units     ║
╠══════════════════╪═════════╪═══════╪═════════════╣
║ Occupancy        │     524 │   0.0 │ 4KiB Blocks ║
║ Free             │ 1297172 │ 100.0 │ 4KiB Blocks ║
║ Clean            │     524 │   0.0 │ 4KiB Blocks ║
║ Dirty            │       0 │   0.0 │ 4KiB Blocks ║
╚══════════════════╧═════════╧═══════╧═════════════╝
```
return '|' not in line # might address that, although I'm not sure if vertical line in this style is the same as the keyboard char

Reply:
Unfortunately, it is not. However I found solution, thanks to Your feedback.